README.md

```diff
@@ -31,7 +31,7 @@ There are two ways to get a source tree:
   Note: cloning GHC from Github requires a special setup. See [Getting a GHC
   repository from Github] [7].
 
-  **DO NOT submit pull request directly to the github repo.**
+  We will accept easily-reviewed documentation changes as GitHub Pull Requests.
```

Comment:
I have just taken what's in the GitHub project description. Whatever we write here, the two should agree.